README.md

```diff
@@ -18,3 +18,10 @@ Content to be unblocked from Cyberoam
 - **Blocked Under**: PersonalStorage
 - **Wikipedia**: Amazon Web Services (AWS) is a subsidiary of Amazon.com that provides on-demand cloud computing platforms to individuals, companies and governments, on a paid subscription basis. The technology allows subscribers to have at their disposal a full-fledged virtual cluster of computers, available all the time, through the Internet.
 - **About**: AWS provides a great set of features to developers for hosting and running applications on servers, but AWS is blocked, making the lives of students a tad more difficult.
+
+### 4. A2 Online Judge
+- **URL**: https://a2oj.com/
+- **Blocked under**: Online Judges.
```

Comment:
Is the actual category called "Online Judges"? I doubt that. The "blocked under" field should include which category it is blocked under.

Reply:
Its showing this in my browser.
![screenshot from 2018-04-24 19-58-16](https://user-images.githubusercontent.com/31186013/39193824-fbac3d26-47f9-11e8-866e-df12d5434884.png)

Reply:
But,if I add it as an exception then it works.So,is it blocked or not?

Reply:
It's not. Somewhere down the line, some certificate is blocked (probably AWS) and that's why this happens unbanning AWS *should* work.

Reply:
Ok Sir..Then, AWS affects a lot of network then..